site/src/pages/TemplateVersionEditorPage/TemplateVersionEditor.tsx

```diff
@@ -582,20 +598,33 @@ export const TemplateVersionEditor: FC<TemplateVersionEditorProps> = ({
 									ref={logsContentRef}
 								>
 									{templateVersion.job.error && (
-										<div>
-											<Alert
-												severity="error"
-												css={{
-													borderRadius: 0,
-													border: 0,
-													borderBottom: `1px solid ${theme.palette.divider}`,
-													borderLeft: `2px solid ${theme.palette.error.main}`,
-												}}
-											>
-												<AlertTitle>Error during the build</AlertTitle>
-												<AlertDetail>{templateVersion.job.error}</AlertDetail>
-											</Alert>
-										</div>
+										<ProvisionerAlert
+											severity="error"
+											title="Error during the build"
+											detail={templateVersion.job.error}
+										/>
+									)}
+
+									{!templateVersion.job.error && !gotBuildLogs && !provisionerDaemonsLoading && (
+										couldntGetProvisioners ? (
+											<ProvisionerAlert
+												severity="warning"
+												title="Something went wrong"
+												detail="Could not determine provisioner status. Your template build may fail. If your template does not build, please contact your administrator"
+											/>
+										) : !compatibleProvisioners || compatibleProvisioners.length === 0 ? (
+											<ProvisionerAlert
+												severity="warning"
+												title="Build Stuck"
+												detail="No Compatible Provisioner Daemons have been configured"
+											/>
+										) : compatibleProvisionersUnhealthy && (
+											<ProvisionerAlert
+												severity="warning"
+												title="Build may be delayed"
+												detail="Compatible Provisioner Daemons have been silent for a while. This may result in a delayed build"
+											/>
+										)
 									)}
```

Comment:
and this is all pretty much the same as the `<ProvisionerAlerts>` from the other file right? could these also be replaced by the new component I suggested?

Reply:
yep. done